cypress/utils/utils.ts

```diff
@@ -368,32 +368,35 @@ export function applySearchFilter(
     cy.wait(4000);
 }
 
-export function sortAsc(
-    sortCriteria: string,
-    tableSelector = `th[data-label="${sortCriteria}"]`
-): void {
-    cy.get(tableSelector).then(($tableHeader) => {
-        if (
-            $tableHeader.attr("aria-sort") === "descending" ||
-            $tableHeader.attr("aria-sort") === "none"
-        ) {
-            $tableHeader.find("button").trigger("click");
-        }
-    });
+//tableSelector should be .pf-c-table__sort as the fieldHeader in views
+export function sortAsc(sortCriteria: string, tableSelector: string): void {
+    cy.get(tableSelector)
+        .contains("th", sortCriteria)
+        .then(($tableHeader) => {
+            const button = $tableHeader.find("button");
```

Comment:
variable name button could be improved to something more specific, such as sortButton, to enhance readability.